Header header
int32 segment_ID
float32 x_pos
float32 y_pos
float32 z_pos
float32 q1_rot
float32 q2_rot
float32 q3_rot
float32 q4_rot
